### PR TITLE
Ensure that the error constants in MediaError are correctly annotated

### PR DESCRIPTION
**Pull request by realityforge**
Constant values extracted from https://developer.mozilla.org/en-US/docs/Web/API/MediaError

This fixes google/elemental2#71

### CODE REVIEW

**Comment by EatingW** [comment]
Created Google internal issue http://b/129275520.

File: externs/browser/html5.js [hunk 1]
```diff
@@ -2386,29 +2386,29 @@ MediaError.prototype.message;
 /**
  * The fetching process for the media resource was aborted by the user agent at
  * the user's request.
- * @type {number}
+ * @const {number}
  */
-MediaError.MEDIA_ERR_ABORTED;
+MediaError.MEDIA_ERR_ABORTED = 1;
```
Comment (EatingW) [inline externs/browser/html5.js]: As Ben mentioned in #3293, we generally avoid putting values in the externs. Otherwise looks good.

Comment (realityforge) [reply externs/browser/html5.js]: Ok - wasn't able to convince anyone that the code size optimization was worth including the values for constants so I have stripped them out and re-pushed the changeset.

**Comment by EatingW** [comment]
Thanks! I'm importing this for internal testing and review.

**Pull request closed without merging**